src/components/Header/index.jsx

```diff
@@ -44,6 +45,9 @@ const Header = ({ darkModeOn, toggleDarkMode }) => {
           <div className="col-auto justify-content-start">
             <Link to="/?course_type=executive-education-2u" className={disableLink('executive-education-2u')}>Executive Education</Link>
           </div>
+          <div className="col-auto justify-content-start">
+            <Link to="/bulk-operation-tasks" className={pathname === '/bulk-operation-tasks' ? disabledLinkClass : ''}>Bulk Operations</Link>
+          </div>
```

Comment:
For now, we should display this only for administrators. This will only be accessible by PCs and not course staff.